### DISCUSSION TITLE
Database export/import not working

### DISCUSSION THREAD

**Discussion by mikesaur** [afourmy/eNMS#342] [Q&A]
I have 2 eNMS test systems built.  1 that has my own routers in it and a fresh new instance.  I did a complete backup via the Admin panel of the one system and imported in to the 2nd system.  Everything completes successfully but the routers from my first instance didn't carry over to the new system.  The device.yaml file does indeed have my routers in it.  Is what I'm doing fully supported?  Any ideas what is going wrong?

**Comment by afourmy** [comment]
> Is what I'm doing fully supported?

Yes, that's how the backup system works.

> Any ideas what is going wrong?

No idea. Can't say without the files. If something does go wrong during the import, there should be a stacktrace in the logs
